edge-e2e/src/main/java/glue/ModuleGlue.java

```diff
@@ -71,7 +73,7 @@ public void connectFromEnvironment(String transportType, Handler<AsyncResult<Con
         try
         {
             ModuleClient client = ModuleClient.createFromEnvironment(protocol);
-            client.open();
+            openClientWithRetry(client);
 
```

Comment:
Figure out what is appropriate time to wait for open before retry

Reply:
I'm going to go with our default exponentialBackoffWithJitter algorithm here